pylxd/tests/test_client.py

```diff
@@ -67,11 +70,18 @@ def raise_exception():
     def test_authentication_failed(self):
         """If the authentication fails, an exception is raised."""
         response = mock.MagicMock(status_code=200)
-        response.json.return_value = {'metadata': {'auth': 'untrusted'}}
+        response.json.return_value = {'metadata': {'auth': 'untrusted',
+                                                   'environment': {}
```

Comment:
You shouldn't need this mock here, as the exception should be raised before we get the metadata.